src/api/mod.rs

```diff
@@ -30,8 +30,14 @@ pub enum ApiError {
     HttpError(#[from] minreq::Error),
     #[error("JSON error")]
     JsonError(#[from] miniserde::Error),
-    #[error("{}", .0.message)]
-    ResponseError(ErrorResponse),
+    #[error("Invalid credentials")]
+    InvalidCredentialsError,
+    #[error("Invalid API key. Log in again.")]
+    ApiKeyError,
+    #[error("Server error: \"{}\"", .0)]
+    ServerError(String),
+    #[error("Client error: \"{}\"", .0)]
```

Comment:
These are the technical names, but do they make sense to the user?

Maybe

```suggestion
    #[error("Internal API error: \"{}\"", .0)]
    ServerError(String),
    #[error("API request failed: \"{}\"", .0)]
```